### PR TITLE
Run i18n normalization & automatic translations for 3.0.0-beta2 

### PR DESCRIPTION
**Pull request by no-reply**
Completed following the process at
https://github.com/samvera/hyrax/wiki/Translations-(internationalization)#update-translations

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ gemspec
 
 group :development, :test do
   gem 'coveralls', require: false
+  gem 'easy_translate'
```
Comment (jcoyne) [inline Gemfile]: Can you add documentation on what this is for and how you use it?

Comment (no-reply) [reply Gemfile]: Does the commit message for 579eb89 satisfy this?

Comment (no-reply) [reply Gemfile]: i.e. there are no changes, except that this dependency used to be added by `i18n-tools`, and now it is not

Comment (jcoyne) [reply Gemfile]: Yes, sorry, I missed that because of the folding.

**Pull request merged**